src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportCreateParquetFilesStep.java

```diff
@@ -44,7 +45,9 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
         workingMap.get(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_BUCKET, GoogleBucketResource.class);
 
     final List<String> paths;
-    if (workingMap.containsKey(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_GSPATHS_FILENAME)) {
+    Boolean exportGsPathsInput =
+        context.getInputParameters().get(JobMapKeys.EXPORT_GSPATHS.getKeyName(), Boolean.class);
+    if (Boolean.TRUE.equals(exportGsPathsInput)) {
```

Comment:
I get why we need to do this to avoid NPEs, but would it be better to make sure `EXPORT_GSPATHS` always has a value instead of doing this? It impacts readability.

Reply:
I think it's better to be null safe when working with wrapper objects than relying on mutating the input somewhere earlier in the logic path

Reply:
I see I'm too late here, but this is case where `Object.requireNonNullElse()` would be better IMO
```
    Boolean exportGsPathsInput =
         context.getInputParameters().get(JobMapKeys.EXPORT_GSPATHS.getKeyName(), Boolean.class);
     if (Objects.requireNonNullElse(exportGsPathsInput, false)) {
```

Reply:
Or to @tlangs 's point, only add use step in flights that have this field defined. If it's not present then it's a fatal flight error, not a silent fallback to `false`.